src/viperleed/utilities/poscar/modify_vacuum.py

```diff
@@ -45,8 +66,19 @@ def modify_vacuum(slab, vacuum_gap_size, absolute=False):
 
     Raises
     ------
-    RuntimeError
-        If the resulting vacuum gap size is negative.
+    NotEnoughVacuumError
+        If the resulting vacuum gap size is negative or too small and 
+        `accept_small_gap` is not enabled in `vacuum_gap_info`.
```

Comment:
```suggestion
        If the resulting vacuum-gap size is negative, or if
        `accept_small_gap` is not enabled in `vacuum_gap_info`
        and the resulting vacuum-gap size is smaller than 5 A.
```

Small rewording suggestion.